data/json/items/book/abstract.json

```
@@ -0,0 +1,238 @@
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "price_post": -100, "intelligence": -1 }
+  },
+  {
+    "abstract": "paperback_western_em",
+    "type": "BOOK",
+    "name": "The Everyman Library",
+    "description": "The Everyman Library is an imprint of Vanilla Media that publishes stories about private eyes, cowboys, quarterbacks, and mobsters.",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "price_post": -100, "intelligence": -1 }
+  },
+  {
+    "abstract": "book_fict_soft_ya_vanilla",
+    "type": "BOOK",
+    "name": "Tween Topics",
+    "description": "Tween Topics is an imprint of Vanilla Media that publishes stories that appeal to the youth of today.  Or, failing that, the parents of said youth.",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "price_post": -400, "intelligence": 1, "fun": -1 }
+  },
+  {
+    "abstract": "book_fict_soft_ya_quiddity",
+    "type": "BOOK",
+    "name": "Quiddity Books",
+    "description": "Quiddity publishes books for young adults.  They offer stories about self-discovery, personal identity, and contemporary trends.",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "price_post": -400, "chapters": 9 }
+  },
+  {
+    "abstract": "book_fict_soft_satire_tpl",
+    "type": "BOOK",
+    "name": "Satire Template",
+    "description": "template for mass produced satirical fiction",
+    "copy-from": "book_fict_soft_tpl",
+    "time": "20 m",
+    "relative": { "intelligence": 3, "price": 249 }
+  },
+  {
+    "abstract": "book_nonf_zine",
+    "type": "BOOK",
+    "name": "Magazine Template",
+    "description": "template for magazine",
+    "intelligence": 4,
+    "weight": "60 g",
+    "volume": "250 ml",
+    "price": 400,
+    "price_postapoc": 100,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "pink",
+    "time": "8 m",
+    "fun": 1
+  },
+  {
+    "abstract": "book_nonf_zine_news_tpl",
+    "type": "BOOK",
+    "name": "News Magazine Template",
+    "description": "template for news magazine",
+    "copy-from": "book_nonf_zine",
+    "time": "11 m",
+    "relative": { "intelligence": 1 }
+  },
+  {
+    "abstract": "book_magazine",
+    "//": "this is the original version of book_nonf_zine",
+    "type": "BOOK",
+    "name": { "str": "readable magazine" },
+    "weight": "70 g",
+    "volume": "250 ml",
+    "material": "paper",
+    "symbol": "?",
+    "color": "pink",
+    "max_level": 1,
+    "intelligence": 5,
+    "time": "8 m",
+    "fun": 1
+  },
+  {
+    "abstract": "book_martial",
+    "type": "GENERIC",
+    "category": "books",
+    "name": { "str": "martial art manual" },
+    "weight": "150 g",
+    "volume": "250 ml",
+    "price": 20000,
+    "material": "paper",
+    "symbol": "?",
+    "color": "green",
+    "use_action": "MA_MANUAL",
+    "time": "10 m",
```

`time` does not work for `GENERIC`, only for `BOOK` items:

```suggestion
    "time": "10 m",
```